app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingManualSignInViewHolder.kt

```diff
@@ -28,28 +28,20 @@ class OnboardingFirefoxAccountViewHolder(private val view: View) : RecyclerView.
         }
     }
 
-    fun bind(autoSignedIn: Boolean) {
-        updateHeaderText(autoSignedIn)
-        updateButtonVisibility(autoSignedIn)
+    fun bind() {
+        updateHeaderText()
     }
 
-    private fun updateButtonVisibility(autoSignedIn: Boolean) {
-        view.turn_on_sync_button.visibility = if (autoSignedIn) View.GONE else View.VISIBLE
-        view.stay_signed_in_button.visibility = if (autoSignedIn) View.VISIBLE else View.GONE
-        view.sign_out_button.visibility = if (autoSignedIn) View.VISIBLE else View.GONE
-    }
-
-    private fun updateHeaderText(autoSignedIn: Boolean) {
-        val icon = if (autoSignedIn) avatarAnonymousDrawable else firefoxAccountsDrawable
+    // TODO move this into the layout?
```

Comment:
Which part - the icon? If it's always going to be this icon I think that makes sense

Reply:
Right, we're not switching icons around anymore. But at the time I wasn't up to the task of adding an icon to the layout code myself :) I think I tried for a minute or two, and quickly gave up @ekager . I shall attempt this again!